test/test.unit.js

```diff
@@ -45,4 +45,5 @@ test('unit: test .handle() and .dispatch()', function(t) {
 	redbloom.dispatch({ns: 'test', cmd: 'w/ payload', payload: 'payload'});
 	redbloom.dispatch({ns: 'test', cmd: 'w/o payload'});
 	redbloom.dispatch({ns: 'test'});
+	redbloom.dispatch({ns: 'cmd'});
```

Comment:
@earayo, you're on the right track that there should be a dispatch here for an action that was not registered.

However, it doesn't seem to test for the expected behaviour that if an unregistered action was dispatch, it should not change the current state.

If you can update this pull request to reflect the expected behaviour, I'll be happy to merge it. Thanks!